src/components/templates/AvatarGrid/RoomModal.tsx

```diff
@@ -86,7 +80,7 @@ export const RoomModal: React.FC<PropsType> = ({
         <h4 className="room-name">{room.title}</h4>
         <div className="room-description">{room.description}</div>
         <div className="room-people">
-          {usersInRoom.map((user, index) => {
+          {recentRoomUsers.map((user, index) => {
             return (
               index + 1 <= MAX_SHOWN_AVATARS && (
                 <div key={index} className={"user"}>
```

Comment:
We should never use `index` as a `key` as it's unstable and causes rendering issues. Can we use `user.id` here as a unique value?
```suggestion
          {recentRoomUsers.map((user, index) => {
            return (
              index + 1 <= MAX_SHOWN_AVATARS && (
                <div key={user.id} className={"user"}>
```

Reply:
Agreed, but this template is legacy AFAIK

Reply:
Yeah, that’s fair.